src/helpers.js

```diff
@@ -105,6 +105,13 @@ function getWindowFromNode(node) {
     throw new Error(
       `It looks like you passed an Array instead of a DOM node. Did you do something like \`fireEvent.click(screen.getAllBy...\` when you meant to use a \`getBy\` query \`fireEvent.click(screen.getBy...\`?`,
     )
+  } else if (
+    node.debug instanceof Function &&
```

Comment:
nit:
```suggestion
    typeof node.debug === 'function' &&
```
works as well? `instanceof` always looks sketchy.